ground_truth_labeling_jobs/video_annotations_quality_assessment/Ground_Truth_Video_Quality_Metrics.ipynb

```diff
@@ -13,6 +13,15 @@
     "Let's start by importing required libraries and initializing session and other variables used in this notebook. By default, the notebook uses the default Amazon S3 bucket in the same AWS Region you use to run this notebook. If you want to use a different S3 bucket, make sure it is in the same AWS Region you use to complete this tutorial, and specify the bucket name for `bucket`. "
```

Comment:
<p>"SageMaker Ground Truth operates using <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-input-data-input-manifest.html" target="_blank">manifests</a>. When you use a modality like image classification, a single image corresponds to a single entry in a manifest and a given manifest contains paths for all of the images to be labeled. Because videos have multiple frames and you can have <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-video-manual-data-setup.html" target="_blank">multiple videos in a single manifest</a>, a manifest is instead organized with a JSON sequence file for each video that contains the paths to frames in Amazon S3. This allows a single manifest to contain multiple videos for a single job.</p><p> </p><p>In this example, the image files are all split out, so you can just grab file paths. If your data is in the form of video files, you can use the Ground Truth console to split videos into video frames. To learn more, see <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-video-automated-data-setup.html" target="_blank">Automated Video Frame Input Data Setup</a>. You can also use other tools like <a href="https://ffmpeg.org/" target="_blank">ffmpeg</a> to split video files into individual image frames. The following block stores file paths in a dictionary."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>